students/10th/soojungsohn/user/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+# Create your models here.
+class Users(models.Model):
+    name = models.CharField(max_length = 50)
+    email = models.CharField(max_length = 50)
+    password = models.CharField(max_length = 300)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        verbose_name = 'Users'
```

Comment:
verbos_name 활용 잘 하셨네요!

테이블의 이름을 지정하는 경우 `verbose_name` 보다는 `db_table`을 사용하고, 이는 모두 소문자로 복수형을 사용합니다.
```suggestion
        db_table = 'users'
```